libraries/AP_Scripting/docs/docs.lua

```diff
@@ -77,7 +77,7 @@ i2c = {}
 ---@param address integer -- device address 0 to 128
 ---@param clock? uint32_t_ud -- optional bus clock, default 400000
 ---@param smbus? boolean -- optional sumbus flag, default false
----@return AP_HAL__I2CDevice_ud
+---@return AP_HAL__I2CDevice_ud|nil
```

Comment:
For documents, it is better to use the word "or" instead of the symbol "|".

Reply:
This is a requirement of the mark up style. 
https://emmylua.github.io/annotations/return.html